index.js

```diff
@@ -1092,6 +1092,47 @@ class GithubScm extends Scm {
         }
     }
 
+    /**
+     * Add a PR comment
+     * @param  {Object}     config
+     * @param  {String}     config.comment     The PR comment
+     * @param  {Integer}    config.prNum       The PR number
+     * @param  {String}     config.scmUri      The SCM URI
+     * @param  {String}     config.token       Service token to authenticate with Github
+     * @return {Promise}                       Resolves when complete
+     */
+    async _addPrComment(config) {
+        const lookupConfig = {
+            scmUri: config.scmUri,
```

Comment:
maybe assign in the beginning `_addPrComment({ comment, prNum, scmUri, token })` so you can shorthand here and don't need to write so many `config.`